index.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v1
+entries:
+  lockc:
+  - apiVersion: v2
+    appVersion: v1.0.0
+    created: "2021-12-07T09:30:27.7876214+01:00"
```

Comment:
I'm happy to merge it how it is now, but since we have timestamps, it would be nice to automate that in the future.